examples/full/main.tf

```diff
@@ -6,81 +6,85 @@ variable "HEADER_FILTER_VALUES" {
   type = list(string)
 }
 
+variable "STRIPE_SECRET_KEY" {
+  type = string
+}
+
 terraform {
   required_providers {
     hookdeck = {
-      source  = "hookdeck/hookdeck"
-      version = "0.5.0-beta.1"
+      source = "hookdeck/hookdeck"
     }
   }
 }
 
 provider "hookdeck" {
-  api_key = var.HOOKDECK_API_KEY
-}
-
-resource "hookdeck_source" "first_source" {
-  name = "first_source"
+  api_key  = var.HOOKDECK_API_KEY
+  api_base = "api.hookdeck.com"
```

Comment:
Note that this shouldn't be required. See https://github.com/hookdeck/terraform-provider-hookdeck/pull/139#issuecomment-2701028721

Reply:
@alexluong 👀 on this, please

Reply:
@leggetter fixed, I pushed a commit to this branch, can you see if that fixes the issue?